File: SignalServiceKit/tests/Contacts/TSGroupThreadTest.m
//
//  Copyright (c) 2020 Open Whisper Systems. All rights reserved.
//

#import "SSKBaseTestObjC.h"
#import "TSGroupThread.h"

NS_ASSUME_NONNULL_BEGIN

@interface TSGroupThreadTest : SSKBaseTestObjC

@end

@implementation TSGroupThreadTest

- (void)testHasSafetyNumbers
{
    TSGroupThread *groupThread = [[TSGroupThread alloc] initWithDictionary:@{} error:nil];
    XCTAssertFalse(groupThread.hasSafetyNumbers);
}

@end

NS_ASSUME_NONNULL_END

File: SignalServiceKit/tests/Contacts/TSGroupThreadTest.swift
//
// Copyright 2022 Signal Messenger, LLC
// SPDX-License-Identifier: AGPL-3.0-only
//

import XCTest
@testable import SignalServiceKit

class TSGroupThreadTest: XCTestCase {
    func testHasSafetyNumbers() throws {
        let groupThread = TSGroupThread.forUnitTest(groupId: Randomness.generateRandomBytes(32))
        XCTAssertFalse(groupThread.hasSafetyNumbers())
    }
}
